tests/units/Model/SubtaskModelTest.php

```diff
@@ -152,9 +152,6 @@ public function testDuplicate()
         $this->assertEquals(0, $subtasks[0]['status']);
         $this->assertEquals(0, $subtasks[1]['status']);
 
-        $this->assertEquals(0, $subtasks[0]['user_id']);
-        $this->assertEquals(0, $subtasks[1]['user_id']);
-
```

Comment:
The unit test should validate that the assignee is correctly copied.

Reply:
Agreed, I'll adapt the unit test again